packages/blade/src/components/Checkbox/__tests__/__snapshots__/CheckboxGroup.web.test.tsx.snap

```diff
@@ -699,7 +699,7 @@ exports[`<CheckboxGroup /> should render with label 1`] = `
 }
 
 .c8:focus + div {
-  outline: 1px solid white;
+  outline: 1px solid hsla(220,30%,96%,1);
```

Comment:
What is this change?

Reply:
So it's the inner outline before the focus ring, previously it was set to white which looked bad on banking dark. 

before: 
<img width="75" alt="image" src="https://github.com/razorpay/blade/assets/35374649/c9447929-b8d2-4e1d-976f-b1b46717f564">


after:  
![Uploading image.png…]()